orca-core/src/main/java/com/netflix/spinnaker/orca/notifications/AbstractPollingNotificationAgent.java

```diff
@@ -17,27 +17,23 @@
 
 import static com.netflix.appinfo.InstanceInfo.InstanceStatus.UP;
 
-import com.google.common.annotations.VisibleForTesting;
 import com.netflix.spinnaker.kork.eureka.RemoteStatusChangedEvent;
-import java.util.concurrent.TimeUnit;
+import io.micrometer.core.instrument.util.NamedThreadFactory;
```

Comment:
sounds like the wrong place to import this

Reply:
what do you mean?

Reply:
it seems weird to pull a util from micrometer

Reply:
in orca core... yeah. true. do you think it's better to duplicate that code (it's ~15 lines)

Reply:
it is clearly gradle imported already though...

Reply:
i copied it from river/cats